routes/users.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import UsersController from '../controllers/users-controller';
+
+const router = express.Router();
+
+/** user registration endpoint */
+router.post('/', UsersController.index);
```

Comment:
#### Nitpick
- Nope, don't call that function index.

Reply:
Noted. I'm looking to use `create` or `register`. Do you find that appropriate?

Reply:
Renamed function to `register`